install/operator/pkg/syndesis/versions/versions.go

```diff
@@ -165,13 +199,135 @@ func APIMigrator(ctx context.Context, c client.Client, n string) (r SyndesisAPIM
 
 func (api syndesisAPI) Migrate() (err error) {
 	if api.v1alpha1 != nil {
-		if err = api.v1alpha1ToV1beta1(); err != nil {
+		if err = api.v1alpha1ToV1beta2(); err != nil {
+			return err
+		}
+	} else if api.v1beta1 != nil {
+		if err = api.v1beta1ToV1beta2(); err != nil {
+			return err
+		}
+	}
+
+	if err = api.updateApis(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// Migrates from old v1alpha1 api to v1beta2. This overwrite v1alpha1
```

Comment:
This scenario is not possible. v1alpha1 is only present for 7.6 and we only support sequential upgrades

Reply:
Covered all possibilities.